### PR TITLE
Add mention of OperatorHub.io to our docs

### PR DESCRIPTION
**Pull request by scholzj**
### Type of change

- Documentation

### Description

We should have a mention in our documentation that Strimzi can be installed using OperatorHub.io. This PR adds this to the assemblies covering Cluster Operator installation. Since the procedure how to install the operator is covered on the OperatorHUb.io website (and might possibly change in the future), it doesn't contain a procedure but just a concept which tells the users to go to the OperatorHub.io website and follow the instructions there.

Unfortunately, there is currently no stable link to the latest version of Strimzi, so it just link to the OperatorHub.io it self.

### CODE REVIEW

File: documentation/book/con-deploying-cluster-operator-using-operator-hub.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// assembly-cluster-operator.adoc
+
+[id='deploying-cluster-operator-using-operator-hub-{context}']
+= Deploying the Cluster Operator using OperatorHub.io
+
+{OperatorHub} is a catalog of Kubernetes operators.
+Operator Hub is using {OperatorLifecycleManager} to install and manage Kubernetes operators.
```
Comment (ppatierno) [inline documentation/book/con-deploying-cluster-operator-using-operator-hub.adoc]: About "Operator Hub", why using an attribute in the sentence before but pure text here?

File: documentation/book/con-deploying-cluster-operator-using-operator-hub.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+You can install stable versions of {ProductName} using OperatorHub.
```
Comment (ppatierno) [inline documentation/book/con-deploying-cluster-operator-using-operator-hub.adoc]: Ditto as above

File: documentation/book/con-deploying-cluster-operator-using-operator-hub.adoc [hunk 3]
```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// assembly-cluster-operator.adoc
+
+[id='deploying-cluster-operator-using-operator-hub-{context}']
+= Deploying the Cluster Operator using OperatorHub.io
```
Comment (laidan6000) [inline documentation/book/con-deploying-cluster-operator-using-operator-hub.adoc]: I think the following title is more accurate:

**Deploying the Cluster Operator from OperatorHub.io**

File: documentation/book/con-deploying-cluster-operator-using-operator-hub.adoc [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
```
Comment (laidan6000) [inline documentation/book/con-deploying-cluster-operator-using-operator-hub.adoc]: _Please consider using the following text:_

{OperatorHub} is a catalog of Kubernetes Operators sourced from multiple providers. It offers you an alternative way to install stable versions of Strimzi using the Strimzi Kafka Operator.

The {OperatorLifecycleManager} is used for the installation and management of all Operators published on {OperatorHub}.

To install Strimzi from {OperatorHub}, locate the _Strimzi Kafka_ Operator and follow the instructions provided.

File: documentation/common/attributes.adoc [hunk 5]
```diff
@@ -80,6 +80,9 @@
 
 :CronExpression: link:http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/tutorial-lesson-06.html[cron expression^]
 
+:OperatorHub: link:https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/[OperatorHub.io^]
```
Comment (laidan6000) [inline documentation/common/attributes.adoc]: The link attribute should go to https://operatorhub.io/

File: documentation/common/attributes.adoc [hunk 6]
```diff
@@ -108,6 +111,7 @@
 // Section enablers
 :Kubernetes:
 :Helm:
+:OperatorHubio:
```
Comment (laidan6000) [inline documentation/common/attributes.adoc]: @PaulRMellor - Please ensure that this new attribute is reflected in #1555

File: documentation/book/con-deploying-cluster-operator-using-operator-hub.adoc (hunk 4 above)
Comment (scholzj) [reply documentation/book/con-deploying-cluster-operator-using-operator-hub.adoc]: Well, you rewrote everything :-o. But your text is better, so I used it.

File: documentation/common/attributes.adoc (hunk 5 above)
Comment (scholzj) [reply documentation/common/attributes.adoc]: I'm stupid :-(

Comment (laidan6000) [reply documentation/common/attributes.adoc]: It's easily done when copying and pasting attributes.

**Pull request merged**